plugins/block-shareable-procedures/src/observable_parameter_model.ts

```diff
@@ -65,6 +63,21 @@ export class ObservableParameterModel
     return this;
   }
 
+  /**
+   * Creates a backing variable in a way that is subclassable.
+   *
+   * @param name The string to set set the variable to.
+   * @param varId The optional ID the backing variable should have.
+   * @returns This parameter model.
+   */
+
+  protected createBackingVariable(name: string, varId?: string): this {
```

Comment:
Sorry I should have asked for this change in my last review :/

Could you actually change this to return the new variable instead of assigning it? Then you can assign it in the constructor. That way all of the side effects are localized to the constructor, and people overriding this function don't have to remember to do the side effect!

Then this should be good to go =)